1_fetch/src/fetch_precip_nc.R

```diff
@@ -3,6 +3,38 @@
 #' @param times the start and end time of the request
 fetch_precip_data <- function(ind_file, view_polygon, times) {
 
+  now <- Sys.time()
+  attributes(now)$tzone <- "UTC"
+  start <- as.POSIXct(times$start, tz = "UTC", format = "%Y-%m-%d")
+  end <- as.POSIXct(times$end, tz = "UTC", format = "%Y-%m-%d")
+  if(start  > now) {
```

Comment:
what if start < now but end > now?

Reply:
Would it be possible to include both historical and forecast precip in the same vizzie? I can imagine that being really compelling partway through a storm